lib/dbManager.php

```diff
@@ -27,22 +27,28 @@ function __construct ($dbFilename, $flags = SQLITE3_OPEN_READWRITE) {
     // @param isAdmin: optionnal, specifies if the user is an admin. Defaults to 0. 
     // @return bool : TRUE if no error, FALSE otherwise
     public function addUser ($username, $password, $gauthSecret, $isAdmin = 0) {
+
+		// Prepare variables before the query
+		$passwordHash = hash('sha256',$password);
         
-        // Prepare variables before the query
-        $username = SQLite3::escapeString ($username);
```

Comment:
Just curious why removing the SQLite3::escapeString function as it supposedly adds extra security with potential characters specific to sqlite queries that might appear in the username while not being already escaped by the previous sanitization performed with htmlspecialchars ?